eng/common/scripts/common.ps1

```diff
@@ -20,7 +21,7 @@ $EngScriptsLanguageSettings = Join-path $EngScriptsDir "Language-Settings.ps1"
 if (Test-Path $EngScriptsLanguageSettings) {
   . $EngScriptsLanguageSettings
 }
-If ($LanguageShort -eq $null)
+if (-not $LanguageShort)
```

Comment:
nit: Perhaps it is just me but I perfer `!$LanguageShort` over `-not $Languageshort`